### PR TITLE
first draft for user_info 

### LINKED ISSUE #34: Add documentation for User Info Cog
https://docs.pydrocsid.ml/cogs/information/user_info/

### PR DESCRIPTION
**Pull request by florian-sabonchi**
closes #34

### CODE REVIEW

File: docs/cogs/information/user_info.md [hunk 1]
```diff
@@ -1 +1,91 @@
...
+## `user`
+The `.user` command shows general information about a given user.
+
+```css
+.[userinfo|user|uinfo|userstats] [user]
+```
+
+|Argument|Required|Description|
+|:------:|:------:|:----------|
+|`user`||The user whose information is requested. If omitted, the requesting user receives their own information via direct messages.|
```
Comment (4norak) [inline docs/cogs/information/user_info.md]:
```suggestion
|`user`||The user whose information is requested. If omitted, the requesting user receives their own information via as a direct message.|
```

File: docs/cogs/information/user_info.md [hunk 2]
```diff
@@ -1 +1,91 @@
...
+
+!!! note
+    Moderation statistics:
+
+    - The term `active` refers to the number of times this user *has actively* reported/punished someone else.
+    - The term `passive` refers to the number of times this user *has been* reported/punished e.g. by a moderator.
+
+
+## `joined`
+The `.joined` command shows a rough estimate of how much time has passed since the last verification of a given member. If no `Verified` role has been configured yet, this estimate refers to the last join date of this member.
```
Comment (4norak) [inline docs/cogs/information/user_info.md:41]: Keep in mind that the actual calculation is probably going to be changed soon, so either remember to update this accordingly or make the documentation more generic here

File: docs/cogs/information/user_info.md [hunk 3]
```diff
@@ -1 +1,91 @@
...
+## `ulog`
+The `.ulog` command shows the moderation log of a given user.
+
+```css
+.[userlogs|userlog|ulog] [user]
+```
+
+|Argument|Required|Description|
+|:------:|:------:|:----------|
+|`user`||The user whose log is requested. If omitted, the requesting user receives their own log via direct messages.|
```
Comment (4norak) [inline docs/cogs/information/user_info.md]:
```suggestion
|`user`||The user whose log is requested. If omitted, the requesting user receives their own log as a direct message.|
```

File: docs/cogs/information/user_info.md [hunk 4]
```diff
@@ -1 +1,91 @@
...
+    - Requesting other users' logs, however, requires the `user_info.view_userlog` permission.
+
+The user log given by this command contains the following events:
+
+- Account creation
+- Guild joins and leaves
+- Username and nickname changes
+- Verification state changes
+- Discord invite approvals and removals
+- Moderation actions:
```
Comment (4norak) [inline docs/cogs/information/user_info.md]:
```suggestion
- Passive moderation actions:
```

**Review by 4norak — CHANGES_REQUESTED** [review]

File: docs/cogs/information/user_info.md [hunk 5]
```diff
@@ -1 +1,91 @@
 # User Info
```
Comment (Tristan-H11) [inline docs/cogs/information/user_info.md:1]:
```suggestion
# User info
```

File: docs/cogs/information/user_info.md [hunk 6]
```diff
@@ -1 +1,91 @@
...
+
+This cog contains some commands to show information about Discord user accounts.
```
Comment (Tristan-H11) [inline docs/cogs/information/user_info.md]:
```suggestion
This cog contains commands to show information about Discord user accounts.
```

File: docs/cogs/information/user_info.md [hunk 7]
```diff
@@ -1 +1,91 @@
...
+
+## `user`
+The `.user` command shows general information about a given user.
```
Comment (Tristan-H11) [inline docs/cogs/information/user_info.md]:
```suggestion
The `.user` command shows general information about the requested user.
```

File: docs/cogs/information/user_info.md [hunk 8]
```diff
@@ -1 +1,91 @@
...
+
+```css
+.[userinfo|user|uinfo|userstats] [user]
+```
+
+|Argument|Required|Description|
+|:------:|:------:|:----------|
+|`user`||The user whose information is requested. If omitted, the requesting user receives their own information via as a direct message.|
```
Comment (Tristan-H11) [inline docs/cogs/information/user_info.md]:
```suggestion
|`user`||The user whose information is requested. If omitted, the requesting user receives their own information via a direct message.|
```

**Review by Tristan-H11 — CHANGES_REQUESTED** [review]

**Review by Tristan-H11 — DISMISSED** [review]

File: docs/cogs/information/user_info.md [hunk 9]
```diff
@@ -1 +1,91 @@
...
+## `user`
+The `.user` command shows general information about the requested user.
+
+```css
+.[userinfo|user|uinfo|userstats] [user]
+```
+
+|Argument|Required|Description|
+|:------:|:------:|:----------|
+|`user`||The user whose information is requested. If omitted, the requesting user receives their own information via a direct message.|
```
Comment (4norak) [inline docs/cogs/information/user_info.md]:
```suggestion
|`user`||The user whose information is requested. If omitted, the requesting user receives their own information as a direct message.|
```

**Review by 4norak — CHANGES_REQUESTED** [review]

**Review by 4norak — APPROVED** [review]

**Pull request merged**